plugins/micromega/sos.ml

```diff
@@ -640,13 +640,13 @@ let deration d =
     let a =
       Q.make
         (foldl (fun a i c -> Z.lcm a (Q.den c)) Z.one (snd l))
-        (foldl (fun a i c -> Z.gcd a (Q.num c)) Z.zero (snd l)) in
+        (foldl (fun a i c -> Z.gcd a (Q.num c)) Z.one (snd l)) in
```

Comment:
This fix is wrong. One is absorbing for GCD. I’ll report the bug upstream. Thanks for tracking it down.

Reply:
https://github.com/ocaml/Zarith/pull/39

Reply:
Oh indeed, you want `gcd 0 n = n`. I suggest we add our own gcd version then for now.

I don't understand what is going it the make-based build system :S

Reply:
In fact I suggest we do _à la_ mathcomp and provide a `\biggcd = gcd_list` that correctly handles this.

Reply:
Rebasing to solve conflict and amending the bad fix.